yarn.lock

```diff
@@ -6416,11 +6416,16 @@ tslib@^1.8.1, tslib@^1.9.0:
   resolved "https://registry.yarnpkg.com/tslib/-/tslib-1.14.1.tgz#cf2d38bdc34a134bcaf1091c41f6619e2f672d00"
   integrity sha512-Xni35NKzjgMrwevysHTCArtLDpPvye8zV/0E4EyYn43P7/7qvQwPh9BGkHewbMulVntbigmcT7rdX3BNo9wRJg==
 
-tslib@^2.0.0, tslib@^2.0.3:
+tslib@^2.0.0:
```

Comment:
Duplicate `tslib` import here, so you might need to run `npx yarn-deduplicate` to fix it.

Reply:
Fixed. I wonder if we need to be checking this in CI.